app/views/sessions/new.html.erb

```diff
@@ -1,2 +1,23 @@
-<h1>Sessions#new</h1>
-<p>Find me in app/views/sessions/new.html.erb</p>
+<div class="container">
+  <div class="row">
+    <div class="col col-lg-4 offset-lg-4 text-center">
+      <h1>Log in</h1>
+      <hr>
+      <div id="login_form">
+        <%= form_for :session, url: new_session_path  do |f| %>
+          <div class="form-group row">
+            <%= f.label :email, class: 'col-sm-4' %>
```

Comment:
Not sure this is how the bootstrap grid is meant to be used. My understanding is it should go on container elements. 

It looks to me like you want a style to change the width of this element